queries.js

```diff
@@ -0,0 +1,64 @@
+const req = require("express/lib/request");
+
+const Pool = require("pg").Pool;
+const pool = new Pool({
+  user: "postgres",
+  host: "localhost",
+  database: "users",
+  password: "mypass",
+  port: 5432,
+});
+const getUsers = (req, res) => {
+  pool.query("SELECT * FROM todo", (error, results) => {
```

Comment:
Please use `async` and `await` for all queries
```suggestion
  pool.query("SELECT * FROM todo", async (error, results) => {
```